[core]/es_extended/server/functions.lua

```diff
@@ -387,6 +387,17 @@ function ESX.SetPlayerFunctionOverride(index)
   Config.PlayerFunctionOverride = index
 end
 
+-- Allows dynamic xPlayer functions, like the overrides but in runtime.
+function ESX.PlayerFunction(name, cb)
```

Comment:
You also need to add the function to the newly created xPlayer in the createExtendedPlayer function

Reply:
whoops, forgot that, will add that :)